docs/blueprints/node-selection.md

```diff
@@ -61,8 +61,7 @@ model node (
 ## Rationale
 
 The Storj Data Science team has currently published two papers on the design of our reputation score:
-[Reputation Scoring](https://github.com/storj/datascience/blob/8b02707dceedd4ce20d699a5a9791ce589b303bd/reputation/Reputation_Scoring_Framework_Highlevel.pdf) and [Extending Audit/Uptime Success Ratios](
-https://github.com/storj/datascience/blob/2ec82c9ec89263d9348798e8a5d50a7b62782110/reputation/extending%20ratios%20to%20reputation/extending%20ratios%20to%20reputation.pdf).  
+[Reputation Scoring](https://github.com/storj/storj/blob/DS_docs/docs/datascience/reputation%20scoring%20framework.pdf) and [Extending Audit/Uptime Success Ratios](https://github.com/storj/storj/blob/DS_docs/docs/datascience/extending%20ratios%20to%20reputation.pdf).  
```

Comment:
The links should be probably relative and link to `docs/datascience`

Reply:
What does a relative link accomplish that this does not? Are they not linked correctly?

Reply:
Currently this links to a `DS_docs` branch rather than the current branch.

Reply:
Note it would also be fine if it's rooted, but yeah... it shouldn't link to a specific branch.

Reply:
Ah fair, will make the change.

Reply:
Links updated